string area_type
Area area
Corridor corridor
Room room
LocalArea[] navigation_areas
Door exit_door